src/modules/blockchainApplication/manage/components/RemoveApplicationDetails/RemoveApplicationDetails.js

```diff
@@ -0,0 +1,169 @@
+  const { t } = useTranslation();
+  const chainId = parseSearchParams(location.search)?.chainId;
+  const { deleteApplicationByChainId } = useApplicationManagement();
+  const { checkPinByChainId, togglePin } = usePinBlockchainApplication();
+  const {
+    name, state, address, lastCertificateHeight, lastUpdated, chainID,
+  } = application.data;
+
+  const isPinned = checkPinByChainId(chainId);
+  const toggleApplicationPin = () => {
+    togglePin(chainId);
+  };
+
+  const footerDetails = [
+    {
+      header: {
+        text: t('Chain ID'),
+        toolTipText: t('The chain ID uniquely identifies a chain in the Lisk ecosystem'),
+      },
+      className: `${styles.detailContentText} ${styles.chainId} chain-id`,
+      content: chainId,
+    },
+    {
+      header: t('Status'),
+      className: `${styles.detailContentText} ${styles.statusChip} ${styles[state]} chain-status`,
+      content: t(state),
+    },
+    {
+      header: t('Last Update'),
+      className: `${styles.detailContentText} last-update`,
+      content: moment(lastUpdated).format('DD MMM YYYY'),
+    },
+    {
+      header: t('Last Certificate Height'),
+      className: `${styles.detailContentText} last-certificate-height`,
+      content: lastCertificateHeight,
+    },
+  ];
+
+  const handleRemoveApplication = () => {
+    deleteApplicationByChainId(chainID);
+    nextStep({ application });
+  };
+
+  return (
+    <Dialog className={`${styles.dialogWrapper} ${grid.row} ${grid['center-xs']}`}>
+      <div className={styles.wrapper}>
+        <Box className={styles.headerContainer}>
+          <p>{t('Remove application')}</p>
+          <div>
+            <img src={chainLogo || liskLogo} />
+          </div>
+        </Box>
+        <Box className={styles.detailsWrapper}>
+          <div className={styles.chainNameWrapper}>
+            <span className="chain-name-text">{name}</span>
+            <TertiaryButton onClick={toggleApplicationPin}>
+              <Icon data-testid="pin-button" name={isPinned ? 'pinnedIcon' : 'unpinnedIcon'} />
+            </TertiaryButton>
+          </div>
+          <Box className={styles.addressRow}>
+            <ValueAndLabel className={styles.transactionId}>
+              <span className="copy-address-wrapper">
+                <CopyToClipboard
+                  text={address}
+                  value={address}
+                  className="tx-id"
+                  containerProps={{
+                    size: 'xs',
+                    className: 'copy-address',
+                  }}
+                />
+              </span>
+            </ValueAndLabel>
+          </Box>
+          <div className={styles.addressRow}>
+            <Link
+              className={`${styles.appLink}`}
+              target="_blank"
+                // eslint-disable-next-line
+                // TODO: this is just a place holder link pending when its part of the response payload from service
+              to={serviceUrl}
+            >
```

Comment:
@eniolam1000752 Please replace this with anchor tag to ensure that it works appropriately